### ISSUE TITLE
beta1.7.0, Some games cannot be saved

### ISSUE THREAD

**Issue by android-1995** [rafaelvcaetano/melonDS-android#829]
When I use version 1.6.1, the game runs normally. But after updating to version 1.7.0, the in-game save file is lost (in fact, it cannot be read). The test game is too big for github to upload, how can I upload it to you?
![image](https://user-images.githubusercontent.com/35674533/154419903-ea098629-ed95-40c0-ab68-47015cea5908.png)

**Comment by android-1995** [comment]
I think it is caused by melonDS updated 0.9.3

**Comment by android-1995** [comment]
Link：https://pan.baidu.com/s/1JXOKgfD8FGkU6idAkloWPA 
Extraction code：1234

**Comment by xxmichibxx** [comment]
AFAIK, older savegames made with previous versions are not compatible. Please review the changelog for more details.

**Comment by android-1995** [comment]
> AFAIK, older savegames made with previous versions are not compatible. Please review the changelog for more details.

No, I'm talking about game saves, not save states.And restart the game without  the recordis not working properly.

**Comment by xxmichibxx** [comment]
Could you upload your ROM to another site? Baidu requires Baidu Netdisk or an account to download.

**Comment by android-1995** [comment]
> Could you upload your ROM to another site? Baidu requires Baidu Netdisk or an account to download.

Can you tell me which site?
